src/components/ViewItem/ViewItem.js

```diff
@@ -136,8 +137,8 @@ class ViewItem extends React.Component {
       'barcode': loan => (<div data-test-item-barcode>{_.get(loan, ['item', 'barcode'])}</div>),
       'dueDate': loan => {
         return (
-          <div data-test-item-due-date>
-            <FormattedDate value={loan.dueDate} />
+          <div data-test-item-due-date className={css.loanDueDate}>
+            <div><FormattedDate value={loan.dueDate} /></div>
```

Comment:
Could you please double check do we need this div?

Reply:
Yes, this div is needed.